src/App.js

```diff
@@ -1,13 +1,38 @@
 import "./App.css";
+import MockmanEs from "mockman-js";
+import { useDispatch } from "react-redux";
 import Header from "./component/LayoutComponent/header/Header";
-import { LandingPage } from "./pages/LandingPage/LandingPage";
-// import { Example } from "./pages/LandingPage/Carousel1";
+import {
+  SidebarLayoutPage,
+  LandingPage,
+  PageNotFound,
+  Explore,
+} from "./pages/index";
+import { fetchVideos } from "./store/Explore/videolisting-actions";
+
+import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
+import { useEffect } from "react";
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchVideos());
+  }, [dispatch]);
   return (
     <div className="App">
-      <Header />
-      <LandingPage />
+      <Router>
+        <Header />
+        <Routes>
+          <Route path="/" element={<LandingPage />}></Route>
+          <Route element={<SidebarLayoutPage />}>
+            <Route path="/explore" element={<Explore />}></Route>
+            <Route path="*" element={<PageNotFound />}></Route>
+          </Route>
+          <Route path="/mockman" element={<MockmanEs />}></Route>
```

Comment:
these Route tags can be auto closing

Reply:
yes i just wanted to be consistent here as i have written some with children route